perllib/FixMyStreet/Script/ArchiveOldEnquiries.pm

```diff
@@ -203,13 +215,16 @@ sub send_email_and_close {
         printf("done.\n    Closing reports: ");
         close_problems($problems);
         printf("done.\n");
+    } elsif ( $email_error && $opts->{show_emails} ) {
```

Comment:
You know email_error is true, otherwise it would have been caught by the first block, so you only need the second half of this if.